src/Pokedex.jsx

```diff
@@ -71,10 +40,10 @@ const Pokedex = (props) => {
   useEffect(() => {
     axios
       .get(`https://pokeapi.co/api/v2/pokemon?limit=807`)
-      .then(function (response) {
-        const { data } = response;
-        const { results } = data;
+      .then((response) => {
+        const { data: { results } = {} } = response;
```

Comment:
You can read more about destructuring nested objects, this was just a small cleanup. The `= {}` is to ensure that the edge case of the object being undefined doesn't break everything.